src/everest/detached/__init__.py

```diff
@@ -446,52 +313,30 @@ def _find_res_queue_system(config: EverestConfig):
     return QueueSystem(queue_system.upper())
 
 
-def generate_everserver_ert_config(config: EverestConfig, debug_mode: bool = False):
-    assert config.config_directory is not None
-    assert config.config_file is not None
-
-    site_config = ErtConfig.read_site_config()
-    abs_everest_config = os.path.join(config.config_directory, config.config_file)
-    detached_node_dir = ServerConfig.get_detached_node_dir(config.output_dir)
-    simulation_path = os.path.join(detached_node_dir, SIMULATION_DIR)
+def generate_everserver_config(config: EverestConfig):
```

Comment:
Should add return type annotation, maybe consider also renaming it something with `queue` in it since it seems to be mostly/only the queue settings for the everserver